### ISSUE TITLE
[Bug]: Invaders do not actually surrender when defeated.

### ISSUE THREAD

**Issue by SigholtStarsong** [igorkis-scrts/A3-Antistasi-Plus#107]
### Describe the bug

When pushed off of the map, the Invaders constantly re-attack from the support corridor area, despite a tooltip informing us that they have 'surrendered' when their last base is removed from the game.

### How to reproduce

Simply push the invaders off the island. Realize you haven't won anything, as they simply reattack.

### Version

2.0.2

### Have you altered the code?

No

### Map

Altis

### What server?

Private dedicated server

### Time bug occured (Server time/UTC)

Repeated occurances

### Mods

```html
CBA_A3	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=450814997
RHSAFRF	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=843425103
RHSUSAF	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=843577117
ace	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=463939057
Advanced Rappelling	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=713709341
Antistasi Plus 2.0	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=2912941775
Zeus Enhanced	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=1779063631
Enhanced Movement	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=333310405
S & S	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=2183975396
RHSGREF	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=843593391
RHSSAF	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=843632231
BackpackOnChest	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=820924072
Blastcore Murr Edition	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=2257686620
CH Bright Nights	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=853333870
CH View Distance	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=837729515
DUI - Squad Radar	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=1638341685
DynaSound 2	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=825181638
Enhanced Movement Rework	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=2034363662
Enhanced Soundscape	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=825179978
F/A-18E/F Super Hornet 2020	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=2131302796
Fire Support Plus	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=2699465073
Freestyles Crash Landing	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=2231150238
Gruppe Adler Trenches	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=1224892496
ILBE Assault Pack - Rewrite	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=1875281645
Immerse	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=825172265
Immersion Cigs	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=753946944
JSRS SOUNDMOD	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=861133494
O&T Expansion Eden	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=1923321700
Reduced Haze Mod v3.1	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=1397683809
Remove stamina	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=632435682
S & S: New Wave	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=2643518676
Task Force Arrowhead Radio (BETA!!!)	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=894678801
TFAR Animations	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=2141020863
WMO - Walkable Moving Objects	Steam	https://steamcommunity.com/sharedfiles/filedetails/?id=925018569
ZEI - Zeus and Eden Interiors	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=1251859358
Zeus Enhanced - ACE3 Compatibility	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=2018593688
RHSPKL	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=1978754337
ACE Compat - RHS AFRF	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=773131200
ACE Compat - RHS USAF	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=773125288
ACE Compat - RHS: GREF	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=884966711
ACE Compat - RHS: SAF	Steam	http://steamcommunity.com/sharedfiles/filedetails/?id=2174495332
```

**Comment by igorkis-scrts** [comment]
Thanks for reporting!
Anyway, it's a known issue, already fixed in the upcoming update.

**Comment by igorkis-scrts** [comment]
Fixed in 2.1.
